h/migrations/versions/2020d79a2d9e_add_annotation_sequence_id_column.py

```diff
@@ -0,0 +1,67 @@
+"""Add the annotation.sequence_id column."""
+import datetime
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.schema import CreateSequence, DropSequence
+
+revision = "2020d79a2d9e"
+down_revision = "64039842150a"
+
+
+Base = declarative_base()
+
+sequence_id_sequence = sa.Sequence(
+    "annotation_sequence_id_seq",
```

Comment:
This is the sequence's name, which is internal to the DB (you'd only use it in future migrations to alter the sequence). It's consistent with the naming style for other sequences, see `tox -qqe dockercompose -- exec postgres psql -U postgres -c "\ds"`